### PR TITLE
Feature/input validation

### PR DESCRIPTION
**Pull request by danmayer**
this adds input validation on the schedule.

### CODE REVIEW

File: src/hundred_pushups/android/core.cljs [hunk 1]
```diff
@@ -201,8 +213,13 @@
             [text {:style {:color "white" :text-align "center" :font-weight "bold"}} "Remove"]]])]]])))
 
 
+(defn invalid-mode []
+  [view {:style {:flex-direction "column" :align-items "center"}}
+   [text {:style {:font-size 20 :font-weight "100" :margin-bottom 10 :text-align "center"}} "Bad user, how did you get here."]])
```
Comment (bhb) [inline src/hundred_pushups/android/core.cljs]: Is this used anywhere?

File: src/hundred_pushups/core.cljc [hunk 2]
```diff
@@ -168,3 +168,9 @@
 
 (defn format-whitelist-row [row]
   (str (name(first row)) ": " (clojure.string/join "-" (last row))))
+
+(def not-nil? (complement nil?))
```
Comment (bhb) [inline src/hundred_pushups/core.cljc]: This is fine, but just FYI, there is a function called [some?](https://clojuredocs.org/clojure.core/some%3F) for this - not your fault, clojure's set of functions around truthiness (and checking for truthiness in a collection) have batshit crazy naming.

**Comment by danmayer** [comment]
yep that wasn't used anymore... bad merge... Updated to `some?` thanks oddly googling let me to add that not-nil function... bad google, couldn't find a better example ;)

**Comment by bhb** [comment]
@danmayer Looks great!

**Pull request merged**